src/tye/Program.PurgeCommand.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using Microsoft.Tye.ConfigModel;
+using Microsoft.Tye.Hosting;
+
+namespace Microsoft.Tye
+{
+    static partial class Program
+    {
+        public static Command CreatePurgeCommand(string[] args)
+        {
+            var command = new Command("purge", "purges state from previous run")
+            {
+                CommonArguments.Path_Required
+            };
+
+            command.Handler = CommandHandler.Create<IConsole, FileInfo>(async (console, path) =>
+            {
+                if (path is null)
+                {
+                    throw new CommandException("No project or solution file was found.");
+                }
+
+                var application = ConfigFactory.FromFile(path);
+
+                using var host = new TyeHost(application.ToHostingApplication(), args);
+                await host.PurgeAsync();
```

Comment:
One of the flaws with this approach is that it's based on reading the previous configuration, and assuming that it's the same, rather than just looking at the `./tye` directory as its source of truth.

That said, it's still an improvement

Reply:
It's basically just ceremonial since it ignores the configuration anyway and just looks at the file in the `./tye` dir.
But I agree that it's not ideal.

Reply:
OK. Let's consider this resolved then for now, this is still an improvment.